tests/TestCase/Http/Middleware/CsrfProtectionMiddlewareTest.php

```diff
@@ -278,10 +278,10 @@ public function testConfigurationCookieCreate()
         $cookie = $response->getCookie('token');
         $this->assertNotEmpty($cookie, 'Should set a token.');
         $this->assertRegExp('/^[a-f0-9]+$/', $cookie['value'], 'Should look like a hash.');
-        $this->assertWithinRange(strtotime('+1 hour'), $cookie['expire'], 1, 'session duration.');
+        $this->assertWithinRange(strtotime('+1 hour'), $cookie['expires'], 1, 'session duration.');
         $this->assertSame('/dir/', $cookie['path'], 'session path.');
         $this->assertTrue($cookie['secure'], 'cookie security flag missing');
-        $this->assertTrue($cookie['httpOnly'], 'cookie httpOnly flag missing');
+        $this->assertTrue($cookie['httponly'], 'cookie httpOnly flag missing');
```

Comment:
These key renames should be added to the migration guide. I don't think rector is going to help us here as we can't easily discern valid replacements.

Reply:
I'll make a PR for the migration guide.